client/src/plus/updates/index.tsx

```diff
@@ -0,0 +1,243 @@
+import Container from "../../ui/atoms/container";
+
+import { useState } from "react";
+import { useSearchParams } from "react-router-dom";
+import useSWR from "swr";
+import { DocMetadata } from "../../../../libs/types/document";
+import { MDN_PLUS_TITLE } from "../../constants";
+import BrowserCompatibilityTable from "../../document/ingredients/browser-compatibility-table";
+import { browserToIconName } from "../../document/ingredients/browser-compatibility-table/headers";
+import { useLocale, useScrollToTop } from "../../hooks";
+import { Button } from "../../ui/atoms/button";
+import { Icon } from "../../ui/atoms/icon";
+import { Loading } from "../../ui/atoms/loading";
+import Mandala from "../../ui/molecules/mandala";
+import { Paginator } from "../../ui/molecules/paginator";
+import BookmarkMenu from "../../ui/organisms/article-actions/bookmark-menu";
+import { NotificationsWatchMenu } from "../../ui/organisms/article-actions/notifications-watch-menu";
+import { useUserData } from "../../user-context";
+import { camelWrap, range } from "../../utils";
+import { Event, Group, useBCD, useUpdates } from "./api";
```

Comment:
Nit: Use two groups, one for package imports (react/react-router-dom/swr), and another one for the local imports:

```suggestion
import { useState } from "react";
import { useSearchParams } from "react-router-dom";
import useSWR from "swr";

import Container from "../../ui/atoms/container";
import { DocMetadata } from "../../../../libs/types/document";
import { MDN_PLUS_TITLE } from "../../constants";
import BrowserCompatibilityTable from "../../document/ingredients/browser-compatibility-table";
import { browserToIconName } from "../../document/ingredients/browser-compatibility-table/headers";
import { useLocale, useScrollToTop } from "../../hooks";
import { Button } from "../../ui/atoms/button";
import { Icon } from "../../ui/atoms/icon";
import { Loading } from "../../ui/atoms/loading";
import Mandala from "../../ui/molecules/mandala";
import { Paginator } from "../../ui/molecules/paginator";
import BookmarkMenu from "../../ui/organisms/article-actions/bookmark-menu";
import { NotificationsWatchMenu } from "../../ui/organisms/article-actions/notifications-watch-menu";
import { useUserData } from "../../user-context";
import { camelWrap, range } from "../../utils";
import { Event, Group, useBCD, useUpdates } from "./api";
```